t/40-openqa-clone-job.t

```diff
@@ -21,14 +21,16 @@ use Test::Exception;
 use Test::More;
 use FindBin;
 use lib "$FindBin::Bin/lib";
-use OpenQA::Test::Utils qw(run_cmd test_cmd);
-
+use Date::Format 'time2str';
+use Mojo::File qw(path curfile tempfile);
+use OpenQA::Test::Utils qw(run_cmd test_cmd stop_service);
+use OpenQA::Test::Database;
 
 sub test_once {
     # Report failure at the callsite instead of the test function
     local $Test::Builder::Level = $Test::Builder::Level + 1;
     # prevent all network access to stay local
-    test_cmd('unshare -r -n script/openqa-clone-job', @_);
+    test_cmd(path(curfile->dirname, '../script/openqa-clone-job')->realpath, @_);
```

Comment:
I prefer to still prevent network access by default to prevent us doing accidental harmful remote actions.